jaeger-core/src/test/java/io/jaegertracing/internal/MDCScopeManagerTest.java

```diff
@@ -0,0 +1,103 @@
+package io.jaegertracing.internal;
+
+import io.jaegertracing.internal.reporters.InMemoryReporter;
+import io.jaegertracing.internal.samplers.ConstSampler;
+import io.opentracing.Scope;
+import io.opentracing.ScopeManager;
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.slf4j.MDC;
+
+import static org.junit.Assert.*;
+
+public class MDCScopeManagerTest {
+
+  InMemoryReporter reporter;
+  JaegerTracer defaultTracer;
+
+  @Before
+  public void setUp() {
+    reporter = new InMemoryReporter();
+    defaultTracer = createTracer(new MDCScopeManager.Builder().build());
+  }
+
+  @Test
+  public void testActiveSpan() {
+    Span mockSpan = Mockito.mock(JaegerSpan.class);
+    try (Scope scope = defaultTracer.activateSpan(mockSpan)) {
+      assertEquals(mockSpan, defaultTracer.activeSpan());
+    }
+
+  }
+
+  @Test
+  public void testDefaultCreation() {
+    Span span = defaultTracer.buildSpan("test Default").start();
+    Scope scope = defaultTracer.activateSpan(span);
+    assertNotNull(MDC.get("traceId"));
```

Comment:
It's better to assert on the actual values

Reply:
added test